README.adoc

```diff
@@ -229,12 +242,31 @@ message in the browser. Otherwise, this method returns the cached property value
 You successfully set up your microservice to have fault tolerance capability.
 
 
+== Enabling metrics for the fault tolerance methods
+
+server.xml
+[source, xml, linenums, role='code_column']
+----
+include::finish/src/main/liberty/config/server.xml[]
+----
+
+MicroProfile Fault Tolerance integrates with MicroProfile Metrics to provide metrics for the annotated fault tolerance methods.
+When both the [hotspot=mpFaultTolerance]`mpFaultTolerance` and the [hotspot=mpMetrics]`mpMetrics` features are included in the [hotspot]`server.xml` configuration file,
+the `@Fallback` fault tolerance annotation supplies metrics that count the total number of annotated method invocations and failed invocations,
+as well, the total number of the fallback method calls.
+
+Note that the [hotspot=mpMetrics]`mpMetrics` feature requires SSL and the configuration has been provided for you. The [hotspot=quickStartSecurity]`quickStartSecurity` and [hotspot=keyStore]`keyStore` configuration elements provide basic security to secure the
+server. When you visit the `/metrics` endpoint, use the credentials defined in the server configuration to log in to view the data for the fault tolerance methods.
```

Comment:
Change to:
"The `mpMetrics` feature requires SSL and the configuration is provided for you. The `quickStartSecurity` and `keyStore` configuration elements provide basic security to secure the
server. When you go to the `/metrics` endpoint, use the credentials that are defined in the server configuration to log in to view the data for the fault tolerance methods."

Reply:
Updated